### ISSUE TITLE
NullReference in MainViewModel

### ISSUE THREAD

**Issue by jibedoubleve** [jibedoubleve/lanceur-bis#93]
# Error
![image](https://user-images.githubusercontent.com/2678545/212707859-b1e74aab-608c-4694-84da-27051f6b8a5e.png)

# Logs
```
2023-01-16 15:56:25.8454;TRACE;Default;1;Fade in Storyboard completed...;
2023-01-16 15:56:25.9125;INFO;Lanceur.Views.MainViewModel;1;Query changed OnNext: Query: ;
2023-01-16 15:56:30.7361;INFO;Default;1;Window deactivated;
2023-01-16 15:56:30.9783;TRACE;Default;1;Fade out Storyboard completed...;
2023-01-16 15:56:41.6626;INFO;Default;1;Window showing;
2023-01-16 15:56:41.9122;TRACE;Default;1;Fade in Storyboard completed...;
2023-01-16 15:56:42.1003;INFO;Lanceur.Views.MainViewModel;1;Query changed OnNext: Query: v;
2023-01-16 15:56:42.1003;TRACE;Default;1;Search: criterion 'v';
2023-01-16 15:56:42.1906;INFO;Default;1;Found plugin 'crash';
2023-01-16 15:56:42.1906;INFO;Default;1;Found plugin 'pdeb';
2023-01-16 15:56:42.2223;INFO;Default;1;Found macro 'DEBUG';
2023-01-16 15:56:42.2223;INFO;Default;1;Found macro 'MULTI';
2023-01-16 15:56:42.2223;INFO;Default;1;Found macro 'STAT';
2023-01-16 15:56:42.2223;INFO;Default;1;Found macro 'CENTRE';
2023-01-16 15:56:42.2223;INFO;Default;1;Found macro 'GUID';
2023-01-16 15:56:42.2223;TRACE;Default;1;Search: criterion 'v' found 4 element(s);
2023-01-16 15:56:42.2223;TRACE;Default;6;Refresh thumbnail of 'C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\devenv.exe'.;
2023-01-16 15:56:42.3455;TRACE;Default;6;Refresh thumbnail of 'C:\Program Files\VideoLAN\VLC\vlc.exe'.;
2023-01-16 15:56:42.3666;INFO;Lanceur.Views.MainViewModel;1;Query changed OnNext: Query: vs;
2023-01-16 15:56:42.3666;TRACE;Default;1;Search: criterion 'vs';
2023-01-16 15:56:42.3666;TRACE;Default;6;Refresh thumbnail of 'C:\Program Files\Oracle\VirtualBox\VirtualBox.exe'.;
2023-01-16 15:56:42.3885;TRACE;Default;7;Refresh thumbnail of 'C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\devenv.exe'.;
2023-01-16 15:56:42.3885;TRACE;Default;1;Search: criterion 'vs' found 1 element(s);
2023-01-16 15:56:43.0281;ERROR;Default;1;Object reference not set to an instance of an object.;"System.NullReferenceException: Object reference not set to an instance of an object.
   at Lanceur.Views.MainViewModel.OnExecuteAliasAsync(ExecutionContext context) in D:\a\lanceur-bis\lanceur-bis\src\Lanceur\Views\MainViewModel.cs:line 212"
2023-01-16 15:56:43.3537;INFO;Lanceur.Views.MainViewModel;1;Query changed OnNext: Query: ;
2023-01-16 15:56:43.6011;TRACE;Default;1;Fade out Storyboard completed...;
2023-01-16 15:56:43.6011;INFO;Default;1;Window deactivated; 
```

**Comment by jibedoubleve** [comment]
Should be investigated for version above 2.1.0 Because stack trace is refering to an old code. If issue doesn't reoccur, then it is fixed

**Comment by jibedoubleve** [comment]
Fixed in version 2.1.0
